### DISCUSSION TITLE
Manage Sales and Discounts through API

### DISCUSSION THREAD

**Discussion by maddo7** [etsy/open-api#1159] [Q&A]
Hello,
is there a way to run/change sales or discounts through the API?
![image](https://github.com/etsy/open-api/assets/17200733/edfb57f3-8598-4d76-b4c6-e5edcd50ad32)

I have read the entire documentation multiple times and couldn't find a way.
Did I miss something?

**Comment by Gareth064** [comment]
You haven't missed anything, there is no way to do this via the API.

**Comment by simonj** [comment]
I Alway want this 😅
